app/controllers/api/v1/events_controller.rb

```diff
@@ -11,7 +11,8 @@ class EventsController < ApplicationController
 
 
       def index
-
```

Comment:
Moeten de require's bovenaan niet in de EventsController staan?

Reply:
Yup ze staan er zo ook in

Reply:
Ja dat zag ik. Ik bedoel dus andersom ;) in de class EventsController (...) end